### DISCUSSION TITLE
Error publishing with pixi in adobe animate on MacBook Pro m1

### DISCUSSION THREAD

**Discussion by G3nzalo** [pixijs/pixijs#7926] [General]
Error when wanting to publish my project with pixi, 
I have installed the pluggin and also electron, but it keeps giving me this error, in macbook pro m1.
  If someone knows how to solve it, it will be of great help,
///
Error al querer publicar mi proyecto con pixi , tengo instalado el pluggin y tambien electron , pero me sigue dando este error , en macbook pro m1 . 
  Si alguien sabe como solucionarlo será de gran ayuda, 

******************************************************************
Error: Electron is required to be installed at usr/local/bin/electron 

Install by running the following from a comandline 
npm install electron - g
******************************************************************

**Comment by miltoncandelero** [comment]
Hola, primero que nada te va a costar conseguir ayuda si no hablas ingles.

Segundo, esto parece un error de electron. deberías probar de ejecutar en una terminal `npm install electron - g`

**Reply by G3nzalo** [reply]
Hola Milton muchas gracias por contestar. 
 Si ya ejecute ese comando , tengo instalado electron.
 También si en la terminal hago electron , me abre electron o electron -v , me dice la version instalada. 
 El problema es al darle publicar en animate , en la terminal de salida de animate sale el mensaje que mencione anteriormente .

**Reply by miltoncandelero** [reply]
Esto es mas un problema entre Animate y la extension https://github.com/pixijs/animate-extension

Lo unico que puedo recomendar y sin mucha idea seria que te asegures de estar usando la ultima version del plugin y que pruebes con distintas versiones de Animate.

Si no, pedi ayuda directamente en el repo de la extension.

**Comment by ivanpopelyshev** [comment]
That's the first M1 issue! Good!
